content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+
+    ```json
+    "endSettings": {
+        "endSettingType": { "date":true },
+        "value": "25 Dec 2021 23:59:00 GMT"
+    }
+    ```
+
+    ```json
+    "endSettings": {
+        "endSettingType": { "amount":true },
+        "value": 10
+    }
+    ```
+
+- `whitelistMintSettings` allows you to configure whitelist settings. Specify the following properites to enable whitelist settings:
+    - `mode` is where you specify whether the whitelist token is burned upon minting.
+        - The `burnEveryTime: true` setting will burn the whitelist token upon mint. Note that the whitelist token must have 0 decimals, otherwise only a partial token will be burned upon minting.
+        - The `neverBurn: true` setting allows whitelist token holders to mint as many times as they wish.
+    - `mint` is the whitelist token mint address
+    - `presale` settings determine if whitelist token holders can mint before the `goLiveDate`
+    - `discountPrice` is an optional discounted price offered to whitelist token holders
+
+    ```json
+    "whitelistMintSettings": {
+        "mode": { "burnEveryTime": true },
+        "mint": "7nE1GmnMmDKiycFkpHF7mKtxt356FQzVonZqBWsTWZNf",
+        "presale": true,
+        "discountPrice": 0.5
+    }
+    ```
+
+
+ - `hiddenSettings` can be used for hide-and-reveal drops (where the image of an NFT is revealed after the mint is complete). Using the `hiddenSettings` is outside the scope of this lesson, but you can read more about it [here](https://docs.metaplex.com/candy-machine-v2/configuration).
+
+    ```json
+    "hiddenSettings": {
+        "name": "My Hidden Collection ",
+        "uri": "uri",
+        "hash": "44kiGWWsSgdqPMvmqYgTS78Mx2BKCWzd"
+    }
+    ```
+
+ - `storage` indicates the storage type to upload images and metadata. This field specifies the service provider that stores the off-chain component of our NFT's data. For this lesson we will use Arweave. Arweave is a decentralized storage network that stores data permanently. Note that Arweave files are only stored for seven days on devnet. If you would like to use Arweave to store your NFT data on mainnet, set the `storage` field to `arweave-sol` instead of `arweave`. You can the review list of supported storage types [here](https://docs.metaplex.com/candy-machine-v2/configuration).
+
+ - `noRetainAuthority` indicates whether the Candy Machine authority has the update authority for each mint or if it is transferred to the minter. This should be kept as `false` for the vast majority of cases. Setting `noRetainAuthority` to `true` would allow the minter of the NFT to change the metadata.
+
+ - `noMutable` indicates whether the NFTs' metadata is mutable or not after minting. If set to false, the metadata can later be updated. If set to true, the metadata cannot be updated later and the setting cannot be reset to false.
```

Comment:
@jamesrp13 
1. awk grammar and apostrophe
2. backtick enclose true and false?

Fix:
 - `noMutable` indicates whether or not the NFT's metadata is mutable after minting. If set to `false`, the metadata can later be updated. If set to `true`, the metadata cannot be updated later and the setting cannot be reset to `false`.